bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/data/ModuleType.java

```diff
@@ -159,7 +159,7 @@ ChannelGroup.BATTERY_EXT, new ChannelGroup(Therm1ChannelHelper.class, GROUP_TYPE
     public final ThingTypeUID thingTypeUID;
     public final FeatureArea feature;
     public final String apiName;
-    public final int thingTypeVersion;
+    public final String thingTypeVersion;
```

Comment:
Why do you need a string here?

Reply:
Later in the code when it is used (injected as a property) it is always converted to String - so no interest to store it as an integer

Reply:
@J-N-K : is it ok for you ? (it is for me)

Reply:
Fine. It should just be made sure that code that sets this property only sets integers (or their string representation).

Reply:
That looks to be the case.